src/sentry/incidents/models.py

```diff
@@ -657,6 +657,35 @@ class Meta:
         unique_together = (("alert_rule", "label"),)
 
 
+class AlertRuleActivationConditionType(Enum):
+    RELEASE_CREATION = 0
+    DEPLOY_CREATION = 1
+
+
+@region_silo_only_model
+class AlertRuleActivationCondition(Model):
```

Comment:
Just curious about code organization -- is there a reason we don't have a `src/sentry/alerts/models.py` where these live? Basically i'm not sure about all these being `incident` models.

Reply:
Yea

This is probably some clean up we should eventually get around to.

everything currently lives under `incident` because alerts used be under incidents/issues :/ 
Now that this has branched out to encompass much more, we should probably split this into a separate `Alerts` directory 🤔

Reply:
`Incident` is basically where all metric alerts code lives. Feel free to rename it, `Incident` should really be called `AlertRuleFire` or something, it's just the table that records when a metric alert fails